.github/workflows/smoke-test-daily.yml

```diff
@@ -39,7 +39,7 @@ jobs:
             - name: Load docker images and start containers.
               working-directory: plugins/woocommerce
               env:
-                  ENABLE_HPOS: 0
+                  ENABLE_HPOS: 1
```

Comment:
If HPOS enabled is default, do we still need this environment variable?

Reply:
Probably not, I just wanted to be explicit about it.